exaudfclient/base/javacontainer/script_options/converter.h

```diff
@@ -15,33 +15,32 @@ namespace SWIGVMContainers {
 
 namespace JavaScriptOptions {
 
-class ScriptOptionsConverter {
+class Converter {
 
     public:
-        ScriptOptionsConverter(std::function<void(const std::string &msg)> throwException,
-                               std::vector<std::string>& jvmOptions);
-
+        Converter();
     
-        void getExternalJarPaths(std::string & src_scriptCode);
+        void convertExternalJar(const std::string & value);
+
+        void convertScriptClassName(const std::string & value);
 
-        void getScriptClassName(std::string & src_scriptCode);
+        void convertJvmOption(const std::string & value);
 
-        void convertImportScripts(std::string & src_scriptCode);
+        const std::set<std::string> & getJarPaths() const {
+            return m_jarPaths;
+        }
 
-        void getExternalJvmOptions(std::string & src_scriptCode);
+        std::vector<std::string>&& moveJvmOptions() {
+            return std::move(m_jvmOptions);
+        }
 
-        const std::set<std::string> & getJarPaths();
     private:
-    
-        std::unique_ptr<ScriptOptionsParser> m_scriptOptionsParser;
-        
-        std::function<void(const std::string &msg)> m_throwException;
 
-        std::vector<std::string>& m_jvmOptions;
+        std::vector<std::string> m_jvmOptions;
         
         std::set<std::string> m_jarPaths;
```

Comment:
Set for jarPaths might change the order of jars. The order of jars might define which classes get loaded, when multiple jars contain classes with the same full name. Did we use a set before?

Reply:
Yes, it was a set before see https://github.com/exasol/script-languages/blob/4ebbbac7f7c0b72ee268f1aa59effb0a3d44c81b/exaudfclient/base/javacontainer/javacontainer_impl.h#L53